generators/language-python-flask/index.js

```diff
@@ -10,7 +10,6 @@ const GENERATE_IMPORT_HERE = "# GENERATE IMPORT HERE";
 const PATH_MAPPINGS_FILE = "./server/config/mappings.json";
 const PATH_LOCALDEV_CONFIG_FILE = "server/localdev-config.json";
 const PATH_REQUIREMENTS_TXT = "./requirements.txt";
-const PATH_PIPFILE = "Pipfile.txt";
```

Comment:
Why not just change this to `Pipfile` instead of `Pipfile.txt` instead of deleting it? Then the changes on 106, 119, and 135 look like they wouldn't be necessary.